src/makeCalculator.js

```diff
@@ -4,7 +4,23 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate: function (sv, x) {
+      this.result = sv(this.result, x);
+
+      return this;
+    },
+    reset: function () {
+      this.result = 0;
+
+      return this;
+    },
+    multiply: (result, x) => result * x,
+    divide: (result, x) => (x === 0 ? result : result / x),
```

Comment:
Similar to the `multiply` method, `divide` should also be a regular function to access the correct `this` context.